### DISCUSSION TITLE
[Other] Upgrading instance installed with shell script

### DISCUSSION THREAD

**Discussion by minorsecond** [jonaswinkler/paperless-ng#925] [Support]
I've installed Paperless-NG via the shell script (docker-compose, etc) and am wondering how to go about keeping the installation up to date. Could I simply add watchtower to the docker-compose.yml file to automatically keep it up to date? Is there a script to run occasionally to update?

**Comment by jonaswinkler** [comment]
See https://paperless-ng.readthedocs.io/en/latest/administration.html#docker-route.

Or watchtower, yes. I believe that providing instructions for watchtower is out of scope for paperless. But since paperless is just another docker image, follow their instructions to get automatic updates.

**Comment by kurti500** [comment]
I have paperless running on my Synology NAS in docker and I have following start script running (as my NAS is not always on and I don't need a daily update task in the background), which updates all running containers 15min after boot (to avoid that docker and the containers are not up and running when the script kicks in):
`sleep 15m
docker run --rm \
    -v /var/run/docker.sock:/var/run/docker.sock \
    containrrr/watchtower \
    --run-once --debug`

As I am quite new to paperless, the paperless container was not updated yet based on above, but it did update today the gotenberg container as well as my jellyfin a few days ago, so I would say it works quite well.

**Reply by minorsecond** [reply]
Nice. That should work. I have had good luck using Watchtower with my other docker containers, so I just added this block into docker-compose.yml:

```
watchtower:
    image: containrrr/watchtower
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - /root/.docker/config.json:/config.json
    command: --interval 86400 --cleanup
```

I figured it would work, but since I installed Paperless using the shell script, I wasn't sure if the docker instructions in the documentation still applied (hence my question).
